deisctl/client/client.go

```diff
@@ -120,13 +121,25 @@ After install, the components will be available to start.
 
 Usage:
   deisctl install [<target>...] [options]
-`
+
+Options:
+  --router-mesh-size=<num>  Number of routers to be loaded when installing the platform [default: %d].
+`, cmd.DefaultRouterMeshSize)
 	// parse command-line arguments
 	args, err := docopt.Parse(usage, argv, true, "", false)
 	if err != nil {
 		return err
 	}
 
+	meshSizeArg, ok := args["--router-mesh-size"].(string)
+	if ok {
```

Comment:
If `!ok` then this would set `RouterMeshSize` to `0`, which then goes to DefaultRouterMeshSize, right? Or are you talking about the case where if someone does "--router-mesh-size=", we may end up with an error.

As a nice touch, I like the fact that we can specify mesh sizes in octal!